Formula/d/dda.rb

```diff
@@ -0,0 +1,46 @@
+class Dda < Formula
+  desc "Tool for developing on the Datadog Agent platform"
+  homepage "https://github.com/DataDog/datadog-agent"
+  url "https://github.com/ofek/pyapp/releases/download/v0.28.0/source.tar.gz"
+  version "0.23.1"
+  sha256 "6baedee5288f2ae7de6537f1c4e92e04caaa642b01ada27edb0b1192c4688073"
+  license any_of: ["Apache-2.0", "MIT"]
+
+  revision 1
```

Comment:
this does not look right?
```suggestion
```

Reply:
Sorry about that, I saw it in another formula. Removed!